runtime/queries/sql/highlights.scm

```diff
@@ -57,7 +57,7 @@
 ] @constant.builtin
 
 ((literal) @constant.numeric
-  (#match? @constant.numeric "^(-?\d*\.?\d*)$"))
+  (#match? @constant.numeric "^-?[0-9]*\.?[0-9]*$"))
```

Comment:
Why the change here? \d seems simpler

Reply:
`\d` is apparently not supported. `\d` won't match where `[0-9]` matches.

Reply:
It's using the rust regex engine and it should be supported

Reply:
Alright. Is it an escaping problem?  I'll give it another quick try...

Reply:
@archseer Here you go:

Current version (correct highlighting):

![Screenshot_20230219_173609](https://user-images.githubusercontent.com/22329650/219961703-6626a847-84ff-4624-8e46-ad4e56ec73b2.png)

Adjusted version, then `:config-reload` (incorrect highlighting of numbers as strings):

![Screenshot_20230219_173720](https://user-images.githubusercontent.com/22329650/219961763-3caa7ff4-f785-49e0-a851-eb0551f58f5c.png)

Reply:
Now surprisingly, this works:

![grafik](https://user-images.githubusercontent.com/22329650/219961807-963864a8-de26-4a98-b1bd-14cd8788b6f1.png)

How was the `\.` not a problem when `\\.` would have been required?